Defra.Cdp.Backend.Api/Endpoints/DeploymentsEndpoint.cs

```diff
@@ -48,6 +49,33 @@ private static async Task<IResult> FindLatestDeployments(IDeploymentsService dep
         );
         return Results.Ok(deploymentsPage);
     }
+    
+    private static async Task<IResult> FindLatestDeploymentsWithMigrations(IDeploymentsService deploymentsService,
+        [FromQuery(Name = "favouriteTeamIds")] string[]? favouriteTeamIds,
```

Comment:
what is the relevance of `favourite`?

Reply:
favourite teams are owned teams. Its the items that get stared in the PFE